src/jit/lsraarmarch.cpp

```diff
@@ -783,23 +780,33 @@ void Lowering::TreeNodeInfoInitPutArgSplit(GenTreePutArgSplit* argNode, TreeNode
 
     if (putArgChild->OperGet() == GT_FIELD_LIST)
     {
-        NYI_ARM("LSRA: Oper for split struct argument is GT_FIELD_LIST");
+        // Generated code:
+        // 1. Consume all of the items in the GT_FIELD_LIST (source)
+        // 2. Store to target slot and move to target registers (destination) from source
+        //
+        argNode->gtLsraInfo.srcCount = argInfo->numRegs + argInfo->numSlots;
```

Comment:
This is probably not (currently) an issue on ARM, but on x86 we had to make the fields (the `argNode` in this case) `regOptional`, because there may not be enough registers to hold them all (up to 8 in the case of 4 fields of `TYP_LONG`. In any case may be something to consider in future to ease register allocation, though on ARM I believe it would require that we have a temporary register available since we can't move directly from spill or lclVar to memory.